script.js

```diff
@@ -0,0 +1,44 @@
+let books = [];
+if (localStorage.books) books = JSON.parse(localStorage.getItem('books'));
+
+const refreshBookList = () => {
+  const booksSection = document.getElementById('books-section');
+  let listItems = '';
+  books.forEach((element, i) => {
+    const book = `
+        <li>
+                    <p>${books[i].title}</p>
+                    <p>${books[i].author}</p>
+                    <button type="button" class="remove" data-index = "${i}">Remove</button>
+                    <hr>
+        </li>
+        `;
+
+    listItems += book;
+  });
+
+  booksSection.innerHTML = listItems;
+
+  document.querySelectorAll('.remove').forEach((element) => element.addEventListener('click', (event) => {
+    const indexBook = event.currentTarget.dataset.index;
+    books.splice(parseInt(indexBook, 10), 1);
```

Comment:
- [ ] **[OPTIONAL]** Instead of using a method like `splice` which mutates the array, I would recommend you use a method like `filter` which only makes a copy of the array.